apps/condo/domains/common/constants/featureflags.js

```diff
@@ -11,6 +11,7 @@ const PROPERTY_BANK_ACCOUNT = 'property-bank-account'
 const PROPERTY_REPORT_DELETE_ENTITIES = 'property-report-delete-entities'
 const SERVICE_PROVIDER_PROFILE = 'service-provider-profile'
 const SHOW_ORGANIZATION_TYPES = 'show-organization-types'
+const ENABLE_DISCOVER_SERVICE_CONSUMERS = 'enable-discover-service-consumers'
```

Comment:
How should we use this flag? Because we wan't to open registration of service providers for any user

Reply:
This flag is related to managing companies.